repos/dde_linux/ports/linux-firmware.port

```diff
@@ -2,7 +2,7 @@ LICENSE   := mixed
 VERSION   := 1
 DOWNLOADS := fw.archive
 
-FW_REV  := b7e07499ca264b5535c5c2b066d8007d92dfe43e
+FW_REV  := b8472dbf35b3a67ad9a00a05182e8ff39400c5c1
```

Comment:
This commit ID is not in https://github.com/cnuke/dde_linux_firmware but the contents should be equivalent to b7e07499ca264b5535c5c2b066d8007d92dfe43e.

@cnuke would you mind to fast-forward your master to make clear what's the canonical head commit?

Reply:
@chelmuth done